### PR TITLE
[v7r3] backported ElasticJobParametersDB from rel-v8r0 branch

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*WMS
NEW: backported ElasticJobParametersDB from rel-v8r0 branch

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py [hunk 1]
```diff
@@ -200,17 +320,32 @@ def deleteJobParameters(self, jobID, paramList=None):
...
             combinedFilter = jobFilter & paramFilter
 
             s = self.dslSearch.query("bool", filter=combinedFilter)
             s.delete()
 
+    def _deleteInNewIndex(self, jobID, paramList):
+        """Deletes a document with this jobID in the new index"""
+        if not paramList:
+            # Deleting the whole record
+            self.log.debug("Deleting record of job {jobID}")
```
Comment (chrisburr) [inline src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py]:
```suggestion
            self.log.debug("Deleting record of job", jobID)
```

**Comment by fstagni** [comment]
There seems to be something weird with the py2 elasticsearch client as it fails in a way that I can't decode. 

Also, py2 pylint prints out weird line numbers ...?!??

**Comment by fstagni** [comment]
Should be OK now. Please review.

**Comment by chrisburr** [comment]
The docs still need to be fixed

**Review by chrisburr — APPROVED** [review]

**Pull request merged**